docs/cheat-sheet.md

```diff
@@ -590,6 +590,34 @@ log_with_returns_run = my_func.run(
 ```
 
 ### Automatic logging
+# Auto logging dataset basic example
```

Comment:
```python
# Auto-logging a dataset: basic example

Reply:
shouldn't it be a sub-sub-sub-sub-header? `####` instead of `#` if the line above has `###`?